src/data/data.h

```diff
@@ -168,7 +194,16 @@ class Data {
         NAV_GNSS_POS_LAT = 0x2B,
         NAV_GNSS_POS_LNG = 0x2C,
         NAV_GNSS_POS_ALT = 0x2D,
-        NAV_GNSS_COURSE = 0x2E
+        NAV_GNSS_COURSE = 0x2E,
+        //TODO deconstruct EVENT and VALVES
+        EVENT_ARMED = 0x2F,
+        EVENT_IGNITED = 0x30,
+        EVENT_CALIBRATED = 0x31,
+        EVENT_SEPERATED = 0x33,
+        EVENT_CHUTE_OPENED = 0x34,
+        EVENT_CHUTE_UNREEFED = 0x35,
+        VALVES = 0x36,
+        CURRENT_STATE = 0x37
```

Comment:
Maybe rename CURRENT_STATE to AV_STATE, as the goat is shared with other SS.
Also the goat will need some cleaning, i.e. data organisation by order of importance.
Maybe drop the explicit enum values as they become more and more impossible to update each time we modify the goat.